activiti-cloud-service-common/activiti-cloud-services-common-security-keycloak/src/main/java/org/activiti/cloud/services/common/security/keycloak/config/CustomKeycloakResolverConfiguration.java

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright 2017-2020 Alfresco Software, Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.activiti.cloud.services.common.security.keycloak.config;
+
+import org.keycloak.adapters.KeycloakConfigResolver;
+import org.keycloak.representations.adapters.config.AdapterConfig;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+@Configuration
+public class CustomKeycloakResolverConfiguration {
```

Comment:
is it in a different class so it is easier to remove when the 11 version gets published?

Reply:
exactly, when we update to 11 we only need to get rid of the class, the configuration where this bean is initialize and clean up the `spring.factories`

Reply:
ok better to have an 11 issue pointing to this code already. Thanks

Reply:
actually, as said here: https://github.com/Activiti/activiti-cloud/pull/160#issuecomment-655582491 , 
that config needs to be a different one, in charge of initializing the bean with the adapter configuration that the CommonSecurityAutoConfiguration uses at startup time